### PR TITLE
TW 788 Update Azure AD configuration page

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 1]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+
+### Mapping group attributes
 
-    > The attributes you select as **Matching** properties are used to match the users in Postman for update operations.
+> Mapping group attributes is optional.
 
-1. (Optional) Enable **Provision Azure Active Directory Groups** by selecting **Yes**.
+To map Postman group attributes to Azure AD group attributes, do the following:
+
+1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Groups** . This is the set of attribute mappings for group objects.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:84]:
```suggestion
1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Groups**. This is the set of attribute mappings for group objects.
```

**Review by mark-anthony-postman — APPROVED** [review]
Changes look great! 🌟

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 2]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+After you set up SCIM in Azure AD, you can configure the integration with Postman for users (required) and for groups (optional).
+
+The attributes you select as **Attribute Mappings** are used to match the users or groups in Postman for update operations.
+
+### Mapping user attributes
+
+To map Postman user attributes to Azure AD user attributes, do the following:
+
+1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Users**. This is the set of attribute mappings for user objects.
+1. (Optional) Under **Target Object Actions**, select **Create**, **Update**, and **Delete**.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:67]: Let us remove `(Optional)` here

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 3]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+1. (Optional) Under **Attribute Mappings**, select **Add New Mapping** to map the following attributes:
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:68]: Let us remove `(Optional)` here

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 4]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+
+    Azure AD attribute | SCIM attribute | Postman attribute | Required
+    --- | --- | --- | ---
+    userPrincipalName | userName | email | TRUE
+    {givenName, surname} | {givenName, familyName} | name | TRUE
+    Switch([IsSoftDeleted], , "False", "True", "True", "False") | Active | active | TRUE
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:74]: Let us change `Active` to `active` in small case

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 5]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+
+### Mapping user attributes
+
+To map Postman user attributes to Azure AD user attributes, do the following:
+
+1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Users**. This is the set of attribute mappings for user objects.
+1. (Optional) Under **Target Object Actions**, select **Create**, **Update**, and **Delete**.
+1. (Optional) Under **Attribute Mappings**, select **Add New Mapping** to map the following attributes:
+
+    Azure AD attribute | SCIM attribute | Postman attribute | Required
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:70]: Let us remove the `Required` column altogether.

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 6]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+
+1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Groups**. This is the set of attribute mappings for group objects.
 1. (Optional) Under **Target Object Actions**, enable **Create**, **Update**, and **Delete**.
 1. (Optional) Under **Attribute Mappings**, select **Add New Mapping** to map the following attributes:
-    | Azure Active Directory Attribute | customappsso Attribute |
-    | ----------- | ----------- |
-    | displayName | displayName |
-    | members | members |
+
+    Azure AD attribute | SCIM attribute | Postman attribute | Required
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:88]: Let us remove the `Required` column altogether.

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 7]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+### Mapping group attributes
 
-    > The attributes you select as **Matching** properties are used to match the users in Postman for update operations.
+> Mapping group attributes is optional.
 
-1. (Optional) Enable **Provision Azure Active Directory Groups** by selecting **Yes**.
+To map Postman group attributes to Azure AD group attributes, do the following:
+
+1. In the Azure AD **Mappings** section, select **Yes** to turn on **Provision Azure Active Directory Groups**. This is the set of attribute mappings for group objects.
 1. (Optional) Under **Target Object Actions**, enable **Create**, **Update**, and **Delete**.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:85]: Let us remove `(Optional)` here

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 8]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
 1. (Optional) Under **Attribute Mappings**, select **Add New Mapping** to map the following attributes:
```
Comment (ArjunSingh-PM) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:86]: Let us remove `(Optional)` here

**Review by ArjunSingh-PM — COMMENTED** [review]
Thanks, Heather. 

I have added a few comments to the document. Kindly review the same.

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 9]
```diff
@@ -53,20 +55,46 @@ Next, you will configure the Azure AD integration.
...
+    Azure AD attribute | SCIM attribute | Postman attribute | Required
+    --- | --- | --- | ---
+    userPrincipalName | userName | email | TRUE
+    {givenName, surname} | {givenName, familyName} | name | TRUE
+    Switch([IsSoftDeleted], , "False", "True", "True", "False") | Active | active | TRUE
 
-1. In the Azure AD **Mappings** section, enable **Provision Azure Active Directory Users**. This is the set of attribute mappings for user objects. Select it to review the attributes that are synchronized from Azure Active Directory to Postman.
+1. Select **Save** to commit any changes.
+
+### Mapping group attributes
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:78]: Yes, I think this is a good plan. Will do!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Comment by akinard-postman** [comment]
Looks good! I did have a question about the various attributes in the group attributes section. Are they considered code and should they use `code` format? It was kind of unclear to me so I thought I would ask for my own reference. 😆

**Review by akinard-postman — APPROVED** [review]
LGTM :)

**Comment by HeatherStenson** [comment]
> I have a question about the various attributes in the group attributes section. Are they considered code and should they use code format? It was kind of unclear to me so I thought I would ask for my own reference.

@akinard-postman Great question! 😁 After some TW team discussion, we agree that even through they're not _exactly_ code they're code-ish and should be formatted accordingly! 🙌

**Comment by akinard-postman** [comment]
@HeatherStenson Thank you for letting me know! :)

**Pull request merged**